docs/content/tutorials/quickstart.md

```diff
@@ -0,0 +1,275 @@
+* `quickstart/*` - files useful for this quickstart.
+
+## Start up Zookeeper
+
+Druid currently has a dependency on [Apache ZooKeeper](http://zookeeper.apache.org/) for distributed coordination. You'll 
+need to download and run Zookeeper.
+
+```bash
+curl http://www.gtlib.gatech.edu/pub/apache/zookeeper/zookeeper-3.4.7/zookeeper-3.4.7.tar.gz -o $zookeeper-3.4.7.tar.gz
+tar xzf $zookeeper-3.4.7.tar.gz
+cd zookeeper-3.4.7
+cp conf/zoo_sample.cfg conf/zoo.cfg
+./bin/zkServer.sh start
+cd ..
+```
+
+## Start up Druid services
+
+With Zookeeper running, we can now start up the Druid processes.
+
+```bash
+java `cat conf-quickstart/druid/historical/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/historical:lib/*.jar io.druid.cli.Main server historical > var/log/historical.log
+java `cat conf-quickstart/druid/broker/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/broker:lib/*.jar io.druid.cli.Main server broker > var/broker.log > var/log/broker.log
+java `cat conf-quickstart/druid/coordinator/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/coordinator:lib/*.jar io.druid.cli.Main server coordinator > var/log/coordinator.log
+java `cat conf-quickstart/druid/overlord/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/overlord:lib/*.jar io.druid.cli.Main server overlord > var/log/overlord.log
+java `cat conf-quickstart/druid/middleManager/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/middleManager:lib/*.jar io.druid.cli.Main server middleManager > var/log/middleManager.log
+```
+
+You should see a log message printed out for each service that starts up. You can view detailed logs
+for any service by looking in the `log/` directory using another terminal.
+
+Later on, if you'd like to stop the services, CTRL-C the supervise program in your terminal. If you
+want a clean start after stopping the services, simply remove the `log/` directory. 
+
+Once every service has started, you are now ready to load data.
+
+## Load example data
+
+We've included a sample of Wikipedia edits from September 12, 2015 to get you started.
+
+```note-info
+This quickstart shows you how to load data in batches, but Druid also supports [ingesting
+streams in real-time](ingestion.html). Druid's realtime ingestion can load data with virtually no
+delay between events occurring and being available for queries.
+```
+
+The [dimensions](https://en.wikipedia.org/wiki/Dimension_%28data_warehouse%29) (attributes you can
+filter and split on) in the Wikipedia dataset, other than time, are:
+
+  * channel
+  * cityName
+  * comment
+  * countryIsoCode
+  * countryName
+  * isAnonymous
+  * isMinor
+  * isNew
+  * isRobot
+  * isUnpatrolled
+  * metroCode
+  * namespace
+  * page
+  * regionIsoCode
+  * regionName
+  * user
+
+The [measures](https://en.wikipedia.org/wiki/Measure_%28data_warehouse%29) (values you can aggregate)
```

Comment:
druid code/config/query-json already has word metric in it in many places, this is introducing another term to the vocab.

you should've chosen that 4 years ago :)